### ISSUE TITLE
Запуск SalesBots на определенную сделку

### ISSUE THREAD

**Issue by nalimovm** [amocrm/amocrm-api-php#521]
Существует ли возможность получить модель SalesBot'ов, а потом запустить одного конкретного бота на одну или несколько сделок?

**Comment by makeroi** [comment]
Да, существует, но через приватное api. Смотрите как это делает интерфейс
